c2_components/src/mfx_c2_encoder_component.cpp

```diff
@@ -897,11 +898,17 @@ mfxStatus MfxC2EncoderComponent::InitEncoder()
             MFX_DEBUG_TRACE_MSG("Encoder initialized");
             MFX_DEBUG_TRACE__mfxStatus(mfx_res);
 
+            // ignore warnings
             if (MFX_WRN_PARTIAL_ACCELERATION == mfx_res) {
                 MFX_DEBUG_TRACE_MSG("InitEncoder returns MFX_WRN_PARTIAL_ACCELERATION");
                 mfx_res = MFX_ERR_NONE;
             }
 
+            if (MFX_WRN_INCOMPATIBLE_VIDEO_PARAM == mfx_res) {
```

Comment:
Is this the root cause of this cts failure?

Reply:
this is one reason of the root cause.